qa/os/src/test/java/org/elasticsearch/packaging/test/PackageTests.java

```diff
@@ -342,10 +341,9 @@ public void test90DoNotCloseStderrWhenQuiet() throws Exception {
             append(tempConf.resolve("elasticsearch.yml"), "discovery.zen.ping.unicast.hosts:15172.30.5.3416172.30.5.35, 172.30.5.17]\n");
 
             // Make sure we don't pick up the journal entries for previous ES instances.
-            clearJournal(sh);
+            String elasticsearchStartTime = sh.run("sleep 2 && date +\"%Y-%m-%d %H:%M:%S\"").stdout.trim();
```

Comment:
Could we just adjust the since time by 2 seconds instead of sleeping?

Reply:
I don't think that would guarantee a gap between Elasticsearch journal messages that's wider than the one-second granularity that the `--since` flag gives us.

If we have test A and test B and both check for the same error message, but test B is broken, we want to make sure that test A and test B's journal messages don't occur in the same second. I'm not sure how to guarantee that except by introducing a pause.

Another approach might be finding a way to tail journal messages to a test-specific file.

Reply:
Some versions of journalctl provide for navigating output with a cursor; I'm going to see if that looks like something we'll have on all platforms.

Reply:
I think I have a solution that avoids invoking `sleep`.